assets/src/dashboard/components/typography/index.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+
+const ExtraExtraLarge = css(
+  ({ theme }) => `
+    font-size: ${theme.typography.presets.xxl.size}px;
+    font-family: ${theme.typography.presets.xxl.family};
+    font-weight: ${theme.typography.weight.normal};
+    line-height: ${theme.typography.presets.xxl.lineHeight}px;
+    letter-spacing: ${theme.typography.presets.xxl.letterSpacing}em;
+
+    @media ${theme.breakpoint.smallDisplayPhone} {
+      font-size: ${theme.typography.presets.xxl.minSize}px;
+      line-height: ${theme.typography.presets.xxl.minLineHeight}px;
+      letter-spacing: ${theme.typography.presets.xxl.minLetterSpacing}em;
+    }
+  `
```

Comment:
certainly not worth changing, but just wanted to mention that the `css` function also has a template string syntax if you ever prefer:
```
const SomeStyles = css`
  ...styles
`
```
I like it for no other reason than you get syntax highlighting. so if you did something like:
```
 const ExtraExtraLarge =  ({ theme }) => css`
    font-size: ${theme.typography.presets.xxl.size}px;
    font-family: ${theme.typography.presets.xxl.family};
    font-weight: ${theme.typography.weight.normal};
    line-height: ${theme.typography.presets.xxl.lineHeight}px;
    letter-spacing: ${theme.typography.presets.xxl.letterSpacing}em;

    @media ${theme.breakpoint.smallDisplayPhone} {
      font-size: ${theme.typography.presets.xxl.minSize}px;
      line-height: ${theme.typography.presets.xxl.minLineHeight}px;
      letter-spacing: ${theme.typography.presets.xxl.minLetterSpacing}em;
    }
  `
```
It should be the equivalent of what you have here, but just add syntax highlighting in your editor

Reply:
Easy enough to update! Thanks for the idea